app/models/form/lettings/questions/managing_organisation.rb

```diff
@@ -1,48 +1,45 @@
 class Form::Lettings::Questions::ManagingOrganisation < ::Form::Question
```

Comment:
We use `answer_options` in `FormController`, and likely a few other places. Do they need adjusting to inject the user and log?